src/pipeline_blocks.py

```diff
@@ -456,10 +456,12 @@ def _application_groupby_agg(config, train_mode, suffix, **kwargs):
 
 
 def _bureau_groupby_agg(config, train_mode, suffix, **kwargs):
+    bureau_cleaned = _bureau_cleaning(config, suffix, **kwargs)
```

Comment:
@pknut if I understand correctly `bureau_cleaned` as other `_cleaned` objects are instanciated twice: in hand-crafted and aggregation features. I think it should be done once and the cleaned object should be passed as function argument. for example here  

```python
 def _bureau_groupby_agg(bureau_cleaned, config, train_mode, suffix, **kwargs):
...
 def _bureau_hand_crafted(bureau_cleaned, config, train_mode, suffix, **kwargs):
```